packages/guides-restructured-text/src/RestructuredText/Parser/Productions/RuleContainer.php

```diff
@@ -49,6 +50,10 @@ public function apply(DocumentParserContext $documentParserContext, CompoundNode
                 $on->addChildNode($newNode);
             }
 
+            if ($production instanceof TitleRule && $on instanceof DocumentNode) {
+                $on->setTitleFound(true);
```

Comment:
Why do we need this?

Reply:
Because field lists items that are parsed before the first title on a page is found create meta-nodes and are NOT output in the document. No FieldlistNode is created.